draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -117,16 +116,23 @@ wherever possible.
 
 ## Feature Summary
 
-* This specification defines
- - a format for the payload of an Issuer-signed JWT containing selectively disclosable claims that include object properties (name-value pairs), array elements, and nested data structures built from these,
- - a format for data associated with the JWT that enables selectively disclosing those claims,
- - facilities for binding the JWT to a key and associated data to prove possession thereof, and
- - a format, extending the JWS Compact Serialization, for the combined transport of the JWT and associated data that is suitable for both issuance and presentation.
-* An alternate format utilizing the JWS JSON Serialization is also specified.
-* This specification enables combining selectively disclosable claims with
-  clear-text claims that are always disclosed.
-* For selectively disclosable claims that are object properties, both the key and value are always blinded.
+This specification defines two primary data formats:
 
+1. SD-JWT is a composite structure issued by the Issuer that consists of the following:
```

Comment:
I'm not sure I'm quite happy with the wording here. We say "... is a composite structure (...) that consists of the following:". I'd then expect to see a list of exactly the things making up the SD-JWT, but I find a list of various formats and alternative formats we define in this spec. What about this?

```suggestion
1. SD-JWT is a composite structure enabling selective disclosure of its contents. The definition in this document comprises the following:
```

Same applies below for SD-JWT+KB

Reply:
I think I agree with @danielfett here.

Reply:
but than maybe makes sense to also clarify line 130 to something like `2. SD-JWT+KB is a composite structure enabling cryptographic key binding when presented to the verifier. The definition in this document comprises of the following:`

Reply:
Yes, as noted in my comment :-) Applied your suggesting with small changes.